test/tests/functional/pbs_cray_smoketest.py

```diff
@@ -144,6 +144,7 @@ def test_cray_compute_job(self):
         scr = []
         scr += ['echo Hello World\n']
         scr += ['sleep 5\n']
+        scr += ['aprun -B sleep 10\n']
```

Comment:
Do you want to assume sleep is in your path or do /bin/sleep?  Not sure which is better.

Reply:
Interesting point.   It's probably safer to have the complete path.

Reply:
Done.